### PR TITLE
added image and callout paragraph styling.

### PR DESCRIPTION
**Pull request by SamTo83**

### CODE REVIEW

File: src/components/Layout/Layout.scss [hunk 1]
```diff
@@ -10,5 +10,7 @@
   // margin-bottom compensates for the height of the fixed navbar
   padding-bottom: 6rem;
   background-color: $background-color;
-  min-height: 100vh;
+  overflow: scroll;
+  overflow-x: hidden;
+  overflow-y: hidden;
```
Comment (DanForder) [inline src/components/Layout/Layout.scss:15]: you've overwritten this file, change it back !

File: src/components/PageHeader/PageHeader.jsx [hunk 2]
```diff
@@ -34,8 +34,8 @@ const PageHeader = (props) => {
             location={location}
           />
           <ImageVideoContainer
+            className="page-header--image"
             content={content}
-            altContent={altContent}
```
Comment (DanForder) [inline src/components/PageHeader/PageHeader.jsx:38]: you've overwritten this file, change it back!

**Review by DanForder — CHANGES_REQUESTED** [review]

File: src/components/PageHeader/PageHeader.jsx (hunk 2 above)
Comment (SamTo83) [reply src/components/PageHeader/PageHeader.jsx:38]: added this back

File: src/components/Layout/Layout.scss (hunk 1 above)
Comment (SamTo83) [reply src/components/Layout/Layout.scss:15]: This is already there.

File: src/components/Paragraph/Paragraph.jsx [hunk 3]
```diff
@@ -1,13 +1,22 @@
...
+      {isCallOut ? (
+        <p data-testid="paragraph" className="paragraph paragraph__callout">
+          {text}
+        </p>
+      ) : (
+        <p data-testid="paragraph" className="paragraph">
+          {text}
+        </p>
+      )}
+    </Fragment>
```
Comment (DanForder) [inline src/components/Paragraph/Paragraph.jsx:19]: don't need a ternary for the whole thing here, the only thing that's different is that one extra class. So could do a ternary just inside that classname

File: src/components/PageHeader/PageHeader.scss [hunk 4]
```diff
@@ -6,4 +6,7 @@
   &--only {
     padding-bottom: 2rem;
   }
+  &--image{
+    border-radius: 15px;
+  }
```
Comment (DanForder) [inline src/components/PageHeader/PageHeader.scss:11]: remove

**Review by DanForder — CHANGES_REQUESTED** [review]

File: src/components/Paragraph/Paragraph.jsx (hunk 3 above)
Comment (noahSerafin) [reply src/components/Paragraph/Paragraph.jsx]: I've sorted this and committed, but I'm not sure if its in the PR

**Review by DanForder — CHANGES_REQUESTED** [review]
run format

**Review by DanForder — APPROVED** [review]

**Pull request merged**